### ISSUE TITLE
Cannot find ASNC folder files

### ISSUE THREAD

**Issue by guilhermemntt** [WindowsNT/AdES#1]
Hello, I cannot find the ASNC folder included files in the solution. Can you provide them?

**Comment by WindowsNT** [comment]
Do you mean the cades.asn1 file?

**Comment by WindowsNT** [comment]
The project includes a binary copy of the ASN.1 compiler if that is what you are looking for.

**Comment by guilhermemntt** [comment]
Yes! I've bypassed the issue setting some properties in my Visual Studio. Now I'm trying to sign a pdf file multiple times using CAdES, but some third parties verification softwares are recognizing five signs when I'm signing twice, for example, and sixteen when I'm signing three times. Do you have any idea of what is happening?

**Comment by WindowsNT** [comment]
Send me the PDF you are trying to sign.

**Comment by guilhermemntt** [comment]
[A17FlightPlan.pdf](https://github.com/WindowsNT/AdES/files/3912924/A17FlightPlan.pdf)

**Comment by WindowsNT** [comment]
I will take a look, thanks.

**Comment by guilhermemntt** [comment]
Any news? Wondering why the PAdES signing doesn't have this issue.

**Comment by WindowsNT** [comment]
What do you mean the PAdES signing doesn't have this issue. PAdES uses CAdES.
Or you meant that you are trying to with the PDF as a binary?

**Comment by guilhermemntt** [comment]
Yes, I've seen your code. I've tried to sign as binary using CAdES but this problem always occurs when signing over a already signed pdf. When I use PAdES (I don't know why) the problem doesn't happen and my verification software recognizes the right number of consecutive signatures.

**Comment by WindowsNT** [comment]
PDF files can techically be signed with plain CAdES (as any file), but the result is not a PDF file and will not be recognized by PDF software. Generally, CAdES signatures only have meaning for software that knows the existance of CAdES - which is not the case for PDF. 

In short, you have to always use PAdES.

**Comment by guilhermemntt** [comment]
I understand, but my pdf viewer still opens my CAdES signed pdf. The problem is the number of signatures identified by the verification software when I sign multiple times. Do you use third party CAdES verification softwares with witch you can test this issue?

**Comment by guilhermemntt** [comment]
I'm currently working at adding consecutive signatures to already signed messages using CryptMsgControl function. Hope send a PR soon. By now, I have the code working, although I prefer refactor some things before send to the repo.
